### ISSUE TITLE
Build failed: You have to use a classifier to attach supplemental artifacts to the project instead of replacing them.

### ISSUE THREAD

**Issue by kerwin-zk** [oap-project/gluten#466]
**Describe the bug**
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-jar-plugin:3.2.0:jar (default) on project spark-sql-columnar-shims-common: You have to use a classifier to attach supplemental artifacts to the project instead of replacing them.

**To Reproduce**
mvn clean package -Pbackends-velox -Pfull-scala-compiler -DskipTests -Dcheckstyle.skip -Dbuild_cpp=ON -Dbuild_velox=ON -Dbuild_velox_from_source=ON -Dbuild_arrow=ON

**Comment by jinchengchenghh** [comment]
You can add -Pspark-3.2 to your compile command, it can work.
But your fix can enable profile spark-3.2 default without specify spark profile which is initial propose.
Thanks for your contribution!
I will merge it until CH backend test pass.

**Comment by FelixYBW** [comment]
let's close it since it's fixed.

**Comment by jinchengchenghh** [comment]
Merged

**Comment by zhztheplayer** [comment]
Reopened this since the changes has been reverted. See more details: https://github.com/oap-project/gluten/pull/477#issuecomment-1295169950

**Comment by jinchengchenghh** [comment]
And this patch will cause Spark33 TPCDS q69 empty result. https://github.com/oap-project/gluten/pull/456
Reproduce:
```
git clone https://github.com/zhztheplayer/gluten-it.git
mvn clean package -Darrow.version=10.0.0-SNAPSHOT -Pspark-3.3 -Pgluten-velox
java -Xmx5G -cp target/gluten-it-1.0-SNAPSHOT-jar-with-dependencies.jar io.glutenproject.integration.tpc.Tpc --backend-type=velox --benchmark-type=ds --fixed-width-as-double --off-heap-size=2g -s=0.1 --cpus=4 --iterations=1 --log-level=2 --queries=q69
```
